sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveParquetSourceSuite.scala

```diff
@@ -206,7 +206,9 @@ class HiveParquetSourceSuite extends ParquetPartitioningTest {
     }
   }
 
-  test("Aggregation attribute names can't contain special chars \" ,;{}()\\n\\t=\"") {
+  // After pushing down aggregate to parquet, we can have something like MAX(C) in column name
+  // ignore this test for now
```

Comment:
In this case, you need to file a JIRA and use an IDed TODO like `TODO(SPARK-XXX)`.

Reply:
I realized that my comment is confusing. I revised it. I was trying to say that we don't have the restriction for `()` any more because of the changes for aggregate push down, so ignore this test for now. Sorry for the confusion.

Reply:
Hmm, still not sure if this will be an issue or not.

Reply:
This test should run OK now